src/components/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../../../api';
+import { Person } from '../../../types';
+import { Loader } from '../../Loader';
+import { PeopleTable } from '../../PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const { slug = '' } = useParams();
```

Comment:
do you need slug here?
if you used it just for passing as a props - delete it, and use from `useParams` in needed component